src/inverseRobot.js

```diff
@@ -24,7 +24,19 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+
+  const keys = Object.keys(robot);
+
+  for (const key of keys) {
+    if (Object.prototype.hasOwnProperty.call(inversedRobot, robot[key])) {
+      return null;
+    } else {
+      inversedRobot[robot[key]] = key;
+    }
```

Comment:
```suggestion
    }
    
    inversedRobot[robot[key]] = key;
```

If you use the `return` keyword within an `if` statement, there is no need to include the `else` branch since it will never be reached.